src/components/List/main/List.js

```diff
@@ -22,17 +24,16 @@ const List = ({ list, index }) => {
           <ListTitle list={list} />
           <Droppable droppableId={list.id}>
             {(provided) => (
-              <div
-                ref={provided.innerRef}
-                {...provided.droppableProps}
-                className="overflow-y-auto overflow-x-hidden"
-              >
-                {cards.map((card, index) => (
-                  <ListCard key={card.id} card={card} index={index} />
-                ))}
-                {provided.placeholder}
-                <AddCard listId={list.id} />
-              </div>
+              <>
+                <div ref={provided.innerRef} {...provided.droppableProps} className="scroll-y-hidden overflow-x-hidden">
+                  {cards.map((card, index) => (
+                    <ListCard key={card.id} card={card} index={index} />
+                  ))}
+                  <AddCardForm listId={list.id} open={open} setOpen={setOpen} />
+                  {provided.placeholder}
+                </div>
+                <AddCardPrompt open={!open} setOpen={setOpen} />
```

Comment:
Something to note here. AddCardForm and AddCardPrompt both share one state value `open` which controls their visibility.
Though they both take opposite values i.e. `open={open}` for AddCardForm, and `open={!open}` for AddCardPrompt.
This allows only one of them to ever be visible at once